src/Brokerage/User.php

```diff
@@ -157,6 +244,95 @@ public function setSelfAccredited($val)
         return $this->_setAttribute("selfAccredited", $val);
     }
 
+    public function setYearsExpAlts($val)

```

Comment:
@jhernan7 , please look through each of these new `set` methods and make sure that validation looks right. These are pretty short, so please spend 5 minutes on it and just make sure it looks sane.